src/lib/components/Toggle/Toggle.js

```diff
@@ -0,0 +1,79 @@
+/*
+ * This file is part of React-SearchKit.
+ * Copyright (C) 2020 CERN.
+ *
+ * React-SearchKit is free software; you can redistribute it and/or modify it
+ * under the terms of the MIT License; see LICENSE file for more details.
+ */
+
+import React, { Component } from 'react';
+import { Checkbox } from 'semantic-ui-react';
+import Overridable from 'react-overridable';
+import _get from 'lodash/get';
+import { Card } from 'semantic-ui-react';
+import _isEmpty from 'lodash/isEmpty';
+import PropTypes from 'prop-types';
+
+class ToggleComponent extends Component{
+    constructor(props) {
+        super(props);
+        this.title = props.title;
```

Comment:
very minor: because of a "bad" habit, we started doing this in the constructor... it is probably not needed, and props can be used directly. Feel free to improve code! :)